packages/shared-components/cypress.config.js

```diff
@@ -46,6 +46,23 @@ module.exports = defineConfig({
           return null;
         }
       });
+
+      on('after:spec', (spec, results) => {
```

Comment:
Isn't this necessary for the workbench cypress tests as well?

Reply:
This is already implemented in e2e-tests/plugins/index.js